addons/test_mail_full/tests/test_portal.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import http
+from odoo.addons.test_mail_full.tests.common import BaseFunctionalTest, TestRecipients
+from odoo.tests.common import HttpCase
+
+
+class TestPortal(BaseFunctionalTest, HttpCase, TestRecipients):
+    def setUp(self):
```

Comment:
Linting: would add a void line before the method

Reply:
Done